packages/web-components/fast-foundation/src/anchored-region/anchored-region.spec.md

```diff
@@ -200,11 +200,16 @@ NOTE: this component api will not be exposed outside of the fast-components pack
 - default slot for content
 
 *functions:*
-- update() - forces the anchored region to recalculate positioning.
+- updateAnchorOffset = (
+    horizontalOffsetDelta: number,
+    verticalOffsetDelta: number
+) 
 
 *Events:*
-- loaded - the region and its contents have been added to the DOM and positioned
-- change - the positioning of the anchored region has changed
+- positionchange
```

Comment:
Suggest we leave in the description text.

```suggestion
- positionchange - the positioning of the anchored region has changed
```

Reply:
yep, and added "loaded" event too